### PR TITLE
Multiple charts

### PR DESCRIPTION
**Pull request by brandoncorey**
This adds the ability to select multiple charts for visualizations. The charts that it enables are line chart, area chart, and bar chart.

On like 95 of `ResultGraph`, there is a workaround to allow bar charts to render correctly when passing data directly to the bar instead of the chart. Normally, the bars will not render correctly, and it is a known issue, but passing an empty array fixes it, but there could be potential side effects, unknown.

### CODE REVIEW

File: src/components/QueryResult.jsx
Comment (AlliLearns) [inline src/components/QueryResult.jsx:1]: It looks like the `stopPropagation` function isn't used.

File: src/components/ResultGraph.jsx [hunk 1]
```diff
@@ -34,8 +37,13 @@ const CustomTooltip = ({ active, payload, label, aggregationType }) => {
   }
 };
 
-const Graph = ({ queryData }) => {
-  const parentRef = useRef(null);
+const Graph = ({ queryData, chartType }) => {
+  console.log(queryData);
```
Comment (AlliLearns) [inline src/components/ResultGraph.jsx]: Please remove `console.log`.

**Review by AlliLearns — CHANGES_REQUESTED** [review]
Looks good, I'm suggesting some small changes.

File: src/components/QueryResult.jsx
Comment (brandoncorey) [reply src/components/QueryResult.jsx:1]: good catch

File: src/components/ResultGraph.jsx (hunk 1 above)
Comment (brandoncorey) [reply src/components/ResultGraph.jsx]: Got it

**Review by AlliLearns — APPROVED** [review]
Excellent, thank you!

**Pull request merged**